content/agent/how-to/run-agent-container.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Running the NGINX Agent in a Container"
```

Comment:
```suggestion
title: "Run the NGINX Agent in a container"
```

We avoid gerunds, and unnecessary capitalisation: a container is a generic term.

Reply:
```suggestion
title: "Running NGINX Agent in a container"
```